### PR TITLE
Update the-ember-times-issue-176.md

### PR DESCRIPTION
**Pull request by MinThaMie**
## Ember Times review template

- [x] Add your name to top and bottom
- [x] Put emoji in writeup title
- [x] Add a short blurb (could be the title) to the beginning
- [x] Link to external article/repo/etc in paragraph/body text, not just the writeup title (link in writeup title is optional now)
- [x] Add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account
- [x] Check that all links work

### CODE REVIEW

File: content/the-ember-times-issue-176.md [hunk 1]
```diff
@@ -31,15 +33,13 @@ Here at the Ember Times, we are super excited about the conference [lineup](http
...
 ---
 
-## [2. Section title in sentence case 🐹](section-url)
+## [The ins and outs of organising EmberConf 📻](https://audio.ember.report/1680877/8115211-003-leah-silber-on-emberconf-2021)
 
-<change section title emoji>
-<consider adding some bold to your paragraph>
-<add the contributor in the post in format "FirstName LastName (@githubUserName)" linked to their GitHub account>
-<please include link to external article/repo/etc in paragraph / body text, not just header title above>
+In the latest [Ember Report](https://twitter.com/EmberReport) [podcast]((https://audio.ember.report/1680877/8115211-003-leah-silber-on-emberconf-2021)) [Leah Silber (@wifelette)](https://twitter.com/wifelette) talks about organising EmberConf remote, the history of EmberConf and the topics like [**inclusion**](https://emberconf.com/about/inclusiveness-at-emberconf) that are so crucial to the EmberConf vibe.
```
Comment (ijlee2) [inline content/the-ember-times-issue-176.md]: I noticed that the link for `podcast` didn't show up. I made a few other suggestions that I think capture the meaning that you had. Can you have a look?

```suggestion
In the latest [Ember Report](https://twitter.com/EmberReport) [podcast](https://audio.ember.report/1680877/8115211-003-leah-silber-on-emberconf-2021), [Leah Silber (@wifelette)](https://twitter.com/wifelette) talks about organising a remote EmberConf, the history of EmberConf, and additional topics like [inclusiveness](https://emberconf.com/about/inclusiveness-at-emberconf) that are so crucial to the EmberConf vibe.
```

File: content/the-ember-times-issue-176.md [hunk 2]
```diff
@@ -31,15 +33,13 @@ Here at the Ember Times, we are super excited about the conference [lineup](http
...
 
-<add your name to author list, top and bottom>
-<add short title to "SOME-INTRO-HERE">
+This podcast gives a nice insight in the mind of a great conference organiser with some tips and pointers for others looking to organise conferences or meet-ups remote and in real live.
```
Comment (ijlee2) [inline content/the-ember-times-issue-176.md]:
```suggestion
This podcast gives a nice insight into the mind of a great conference organiser. There are also tips and pointers for people looking to organise conferences or meet-ups remotely and in real life.
```

File: content/the-ember-times-issue-176.md [hunk 3]
```diff
@@ -31,15 +33,13 @@ Here at the Ember Times, we are super excited about the conference [lineup](http
...
+
+Besides information on how to organise a conference Leah also tells us about her favourite talks and the cool things that will happen this EmberConf including workshops and the [kids track](https://emberconf.com/kids).
```
Comment (ijlee2) [inline content/the-ember-times-issue-176.md]:
```suggestion
In addition to how to organise a conference, Leah tells us about her favourite talks and the cool things that will happen at EmberConf 2021, such as workshops and the [kids track](https://emberconf.com/kids).
```

After this paragraph, what do you think about adding a call to action? Something like:

```
Be sure to check out the podcast episode, [Leah Silber on EmberConf 2021](https://audio.ember.report/1680877/8115211-003-leah-silber-on-emberconf-2021)!
```

**Review by ijlee2 — APPROVED** [review]
Thank you for letting us know what we can expect in the podcast episode with Leah. I enjoyed listening to the episode and think more will thanks to your writeup!

**Comment by amyrlam** [comment]
rebased

File: content/the-ember-times-issue-176.md (hunk 1 above)
Comment (amyrlam) [reply content/the-ember-times-issue-176.md]: Looks good, thank you both! I'm hoping to submit a bit earlier today than normal, so am going to merge in these changes while have some cycles.

**Pull request merged**